src/main/scala/org/constellation/infrastructure/configuration/CliConfigParser.scala

```diff
@@ -44,17 +44,35 @@ object CliConfigParser {
       opt[Unit]('t', "test-mode")
         .action((x, c) => c.copy(testMode = true))
         .text("Run with test settings"),
+      opt[String]('k', "keystore")
```

Comment:
Try to use application.conf instead so the parsing, validation is done automatically values can be pass as java properties e.g `java -Dconstellation.keystore.path`

Reply:
but this will need to be provided by the node operator during the startup, especially I'll not provide my passwords into `application.conf` ;)

Reply:
Sure but path and other thing can be, plus it's not about putting it in the conf file but get right of opt handling parsing